vertical-pod-autoscaler/pkg/updater/priority/update_priority_calculator_test.go

```diff
@@ -336,37 +336,131 @@ func TestDontUpdatePodWithOOMAfterLongRun(t *testing.T) {
 	assert.Exactly(t, []*apiv1.Pod{}, result, "Pod shouldn't be updated")
 }
 
-func TestDontUpdatePodWithOOMOnlyOnOneContainer(t *testing.T) {
-	calculator := NewUpdatePriorityCalculator(
-		nil, nil, &UpdateConfig{MinChangePriority: 0.5}, &test.FakeRecommendationProcessor{})
-
-	pod := test.Pod().WithName("POD1").AddContainer(test.BuildTestContainer(containerName, "4", "")).Get()
-
-	// Pretend that the test pod started 11 hours ago.
-	timestampNow := pod.Status.StartTime.Time.Add(time.Hour * 11)
-
-	pod.Status.ContainerStatuses = []apiv1.ContainerStatus{
+func TestDontUpdatePodWithOOM_VpaOvservedContainers(t *testing.T) {
```

Comment:
TestDontUpdateModWithOOM is no longer a good name for this testcase

Reply:
Changed to `TestQuickOOM".